components/camel-jooq/pom.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>components</artifactId>
+        <groupId>org.apache.camel</groupId>
+        <version>3.0.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>camel-jooq</artifactId>
+    <packaging>jar</packaging>
+    <name>Camel :: JOOQ</name>
+    <description>Camel JOOQ support</description>
+
+    <properties>
+        <org.jooq.version>3.11.9</org.jooq.version>
```

Comment:
This property placeholder can be in parent/pom

Reply:
Thanks, fixed